README.md

```diff
@@ -3,8 +3,43 @@
 If you are the kind of person who likes thinking _long-term_ and _high-impact_
 this repo is for you!
 
-**D**o **W**hat **Y**ou **L**ove is _not_ a 3-month stepping-stone project ...
+**D**o **W**hat **Y**ou **L**ove is _not_ a stepping-stone project, we're in it for the long-haul. And after a year and a half, we're making sure that the direction we're heading in is clear to everyone who wants to know.
 
-We _very much_ want/need help on this so if you have _time_
+### Our Goal/Mission
+
+> Empower people to sustainably unlock their potential and pursue their passion through technology.
```

Comment:
Pursuing passions is also something which is quite controversial advice

http://www.forbes.com/sites/robasghar/2013/04/12/five-reasons-to-ignore-the-advice-to-do-what-you-love/2/#43dd32812e4c
http://uk.businessinsider.com/you-should-love-what-you-do-not-do-what-you-love-2015-12
http://time.com/money/3990452/tokumitsu-do-what-you-love/
http://qz.com/259190/why-do-what-you-love-is-bad-career-advice/